src/store.ts

```diff
@@ -43,7 +43,7 @@ export class TypeormStore extends Store {
   private readonly expirationInterval: number;
   private expirationIntervalId?: number;
 
-  constructor(options: Options) {
+  constructor(options: any) {
```

Comment:
This seems like a bad thing. Options for the store should be properly typed.

Reply:
Typed it better as it was missing some typing